Objects/longobject.c

```diff
@@ -4080,7 +4225,7 @@ long_mod(PyObject *a, PyObject *b)
         return fast_mod((PyLongObject*)a, (PyLongObject*)b);
```

Comment:
Couldn't this be removed?

Reply:
It's maintained for performance purposes.

Reply:
but it's duplicate of the code in `l_mod`

Reply:
The whole function can actually be removed altogether.

Reply:
Yeah, we could probably remove that.